run_code.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
This platform dependent so needs to be clarified. Perhaps something for an "examples" directory.